cfgov/unprocessed/js/routes/common.js

```diff
@@ -19,7 +19,6 @@ require( '../modules/scroll-on-history-collapse' ).init();
 require( '../modules/clear-form-buttons' ).init();
 require( '../modules/UStreamPlayer' ).init( '.video-player__ustream' );
```

Comment:
Can livestreaming appear outside of an events page? Does this need to be global?